containers/themes/ThemesSection.js

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import { c } from 'ttag';
+import {
+    useMailSettings,
+    useEventManager,
+    useApiWithoutResult,
+    SubTitle,
+    Alert,
+    ThemeCards,
+    ConfirmModal,
+    TextArea
+} from 'react-components';
+import { updateTheme } from 'proton-shared/lib/api/mailSettings';
+import { themeDark, themeLight, themeBlue, themeCustom } from './availableThemes.js';
+
+const availableThemes = [themeDark, themeLight, themeBlue, themeCustom];
+
+const getThemeName = (Theme) => {
+    if (Theme) {
+        if (/\/* light-theme \/*/.test(Theme)) {
+            return 'light-theme';
+        }
+        if (/\/* blue-theme \/*/.test(Theme)) {
+            return 'blue-theme';
+        }
+        return 'custom-theme';
+    }
+    return 'dark-theme';
+};
+const getCSS = (themeName) => {
+    switch (themeName) {
+        case 'light-theme':
+            return CSSLightTheme;
+        case 'blue-theme':
+            return CSSBlueTheme;
+        default:
+            return '';
+    }
+};
+
+const ThemesSection = () => {
+    const [showCustomModal, setShowCustomModal] = useState(false);
```

Comment:
Use `useModal` hook for that.

Reply:
Yep, the code looks way less cluttered now